include/faabric/rpc/RPCServer.h

```diff
@@ -27,5 +27,7 @@ class RPCServer
     std::thread servingThread;
 
     virtual void doStart(const std::string& serverAddr) = 0;
+
+    virtual void doStop() = 0;
```

Comment:
It seems that every subclass calls `server->Shutdown()`, so rather than making this a pure virtual function, I would just make it virtual, provide a default implementation which calls `server->Shutdown()`, then override in the async one.